### PR TITLE
Added tooltip when product is already added in cart #7727

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

File: templates/webshop/base.jinja [hunk 1]
```diff
@@ -466,8 +466,14 @@
         $.each(lines, function(index, line) {
           $(".product-" + line.product.id + "-buy-now-btn")
             .attr('disabled', 'disabled');
+            var disable_check=$('.product-buy-now').prop('disabled')
```
Comment (tarunbhardwaj) [inline templates/webshop/base.jinja]: `;`

File: templates/webshop/base.jinja [hunk 2]
```diff
@@ -466,8 +466,14 @@
         $.each(lines, function(index, line) {
           $(".product-" + line.product.id + "-buy-now-btn")
             .attr('disabled', 'disabled');
+            var disable_check=$('.product-buy-now').prop('disabled');
```
Comment (GauravButola) [inline templates/webshop/base.jinja]: isDisabled?

**Comment by GauravButola** [comment]
LGTM.

**Comment by gautampanday** [comment]
add-cart-top

File: templates/webshop/base.jinja [hunk 3]
```diff
@@ -558,15 +567,10 @@
...
-            text: 'Product Added to your Cart',
-            history: false,
-            nonblock: true,
-            delay:2000,
-            nonblock_opacity: .4,
-            animation: 'show'
+        .done(function(result){
+          $(".cart-message").removeClass("hidden").show().delay(3000).fadeOut();
+          $(".cart-add-msg").html(function() {
+            return '<div class="col-sm-5">' + '<img class="img-responsive"' + "src=" + result.line.image + ">" + "</div>" + '<div class="col-sm-7">' + "<p>" + result.line.display_name + "<br>" + "added to your cart" + "</p>" + "</div>";
```
Comment (tarunbhardwaj) [inline templates/webshop/base.jinja]: Use underscore js template. which is already being used in rendering cart popup

File: templates/webshop/product.jinja [hunk 4]
```diff
@@ -142,14 +142,15 @@
...
             <form action="{{ url_for("nereid.cart.add_to_cart") }}" method="post" id="product-buy-now" class="add-to-cart" autocomplete="off" data-ga-event-label="Add To Cart" data-ga-product-name="{{ product.name }}" data-ga-product-category="{{ product.category and product.category.name or None }}" data-ga-product-price="{{ product.sale_price()|currencyformat(request.nereid_currency.code) }}">
-              <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-              <input type="hidden" name="quantity" value="1"/>
-              <input type="hidden" name="size" id="size-id"/>
-              <input type="hidden" name="product" value="{{ product.id }}"/>
-              <input type="hidden" name="action" value="add"/>
-              <button class="btn btn-primary btn-buynow btn-block btn-lg product-buy-now product-{{ product.id }}-buy-now-btn" type="button" {% if not product.can_buy_from_eshop() %}disabled{% endif %}
-              quantity="1" id="buy-now-btn">Add to Cart</button>
-
+              <div class="tooltip-bottom" data-placement="bottom" title="This product is already added to the cart">
```
Comment (tarunbhardwaj) [inline templates/webshop/product.jinja]: this tooltip will be always be there for all popup

File: templates/webshop/base.jinja [hunk 5]
```diff
@@ -558,16 +578,11 @@
...
-            history: false,
-            nonblock: true,
-            delay:2000,
-            nonblock_opacity: .4,
-            animation: 'show'
-          });
+        .done(function(result){
+          $(".cart-message").removeClass("hidden").show();
+          var template = $("#cart-message").html();
+          $(".cart-add-msg").html(_.template(template, {'result': result}));
```
Comment (tarunbhardwaj) [inline templates/webshop/base.jinja]: `line`: result.line

**Pull request merged**